anitya/__init__.py

```diff
@@ -145,7 +145,8 @@ def log(session, project=None, distro=None, topic=None, message=None):
                              'project: %(project)s',
         'project.edit': '%(agent)s edited the project: %(project)s fields: '
                         '%(changes)s',
-
+        'project.flag': '%(agent)s flagged the project: %(project)s',
```

Comment:
I think I would like to have it in yes. It'll give the admin an idea what this flag is about when looking at the list of flags.